nemo/collections/asr/data/audio_to_text.py

```diff
@@ -1341,8 +1341,7 @@ def __next__(self):
             try:
                 sample = next(self.wrapped_iter)
             except StopIteration:
-                self.wrapped_iter = iter(self.wrapped_ds)
```

Comment:
Why is this being done ?

Reply:
This fix was done because bucketing datasets will hang if any number of samples in the buckets do not divide properly by world size.